core/arch/riscv64/riscv64.asm

```diff
@@ -60,33 +60,31 @@ GLOBAL_LABEL(cpuid_supported:)
         DECLARE_FUNC(call_switch_stack)
 GLOBAL_LABEL(call_switch_stack:)
         /* Init the stack. */
-        addi     sp, sp, -32
-        /* Use two callee-save regs to call func. */
-        sd       ra, 24 (sp)
-        sd       s0, 16 (sp)
-        sd       s1, 8 (sp)
-        sd       s2, 0 (sp)
+        addi    sp, sp, -32
```

Comment:
The alignment of the start of the operands is different to other code styles.

Reply:
It aligned to 8 spaces.

Reply:
I mean the operands, not the operators. For example:
```
        addi    sp, sp, -32
        ldp      x20, x30, [sp], #32
        ldp      x20, x30, [sp], #32
        addi    sp, sp, -32
```

Reply:
Yes, the first operand is also aligned to 8 spaces. ARM and X64 ports are aligned to 9 spaces, which is strange. I don't think we have to follow that unless there is a reason. 8 spaces are more convenient for typing.

Reply:
The style convention is documented at https://dynamorio.org/page_code_style.html#autotoc_md126:
```
16. In .asm files, place opcodes on column 8 and start operands on column 17.
```
Other ports are following that convention: even if you find some exception in some old code, please follow the documented style conventions to keep all new code consistent.

Reply:
Oh OK, didn't noticed that.

Reply:
Done. @derekbruening Could you review more?